app.js

```diff
@@ -0,0 +1,53 @@
+class Books {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  displayBooks() {
+    const booksList = document.getElementById('books-list');
+    booksList.innerHTML = '';
+    this.books.forEach((book) => {
+      const li = document.createElement('li');
+      li.classList.add('book');
+      li.innerHTML = `<div>"${book.title}" by ${book.author}</div>`;
+      const removeButton = document.createElement('button');
+      removeButton.innerHTML = 'Remove';
+      removeButton.addEventListener('click', () => {
+        this.removeBook(book.title);
+      });
+
+      li.appendChild(removeButton);
+      booksList.appendChild(li);
+      li.style.listStyleType = 'none';
+      const hr = document.createElement('hr');
+      booksList.appendChild(hr);
+    });
+  }
+
+  addBook(title, author) {
+    const book = { title, author };
+    this.books.push(book);
+    localStorage.setItem('books', JSON.stringify(this.books));
+    this.displayBooks();
+  }
+
+  removeBook(title) {
+    const newBooks = this.books.filter((book) => book.title !== title);
+    localStorage.setItem('books', JSON.stringify(newBooks));
+    this.books = newBooks;
+    this.displayBooks();
+  }
+}
```

Comment:
- [ ] You did a great job refactoring your project to use ES6 classes! 🚀 However, according to the general project requirements:

![image](https://user-images.githubusercontent.com/77069011/227046224-acd3a9fb-fd7b-4793-8521-a08a792aaa74.png)

If I create two books with the same title (which I'm allowed to do so) both of them are removed, I suggest that you either include a check to avoid having duplicate titles or include something like a unique ID (it could be the date in ms) which lets you identify the one book you need to remove. 🤓 🔧